daisy/compute/compute.go

```diff
@@ -217,7 +217,7 @@ func (c *client) InstanceStopped(project, zone, name string) (bool, error) {
 	switch status {
 	case "PROVISIONING", "RUNNING", "STAGING", "STOPPING":
```

Comment:
Stopping should actually never be hit as I think that only would happen if you call a stop. Shutdown from inside the VM would just go from Running to Stopped (I think)